spec/requests/api/stateless/transactions_spec.rb

```diff
@@ -6,10 +6,31 @@
 
   let!(:member) { create(:member, email: 'test@example.com') }
   let!(:customer) { create(:payment_braintree_customer, member: member) }
-  let!(:subscription) { create(:payment_braintree_subscription, customer: customer) }
-  let!(:subscription_transaction) { create(:payment_braintree_transaction, subscription: subscription, customer: customer ) }
-  let!(:standalone_transaction_a) { create(:payment_braintree_transaction, customer: customer ) }
-  let!(:standalone_transaction_b) { create(:payment_braintree_transaction, customer: customer ) }
+  let!(:payment_method) { create(:braintree_payment_method,
+    customer: customer,
+    id: 1432,
+    instrument_type: 'credit card',
+    token: '2ewruo4i5o3',
+    last_4: '2454',
+    email: customer.email,
+    card_type: 'Mastercard')}
+  let!(:subscription) { create(:payment_braintree_subscription,
+    customer: customer,
+    payment_method: payment_method) }
+  let!(:subscription_transaction) { create(:payment_braintree_transaction,
```

Comment:
I'm not sure what you mean with re-testing activerecord. Transactions either have a subscription id or not. If you get all transactions for a customer, you also get the ones that are associated with subscriptions. The endpoint is specifically designed to return only standalone transactions, which is why it's necessary to have a spec that demonstrates that transactions that are associated with subscriptions won't be returned. This isn't something that's covered by model specs.